src/visu.f90

```diff
@@ -96,7 +97,9 @@ subroutine visu_init()
     endif
     if (iscalar.ne.0) then
        do is = 1, numscalar
-          call decomp_2d_register_variable(io_name, "phi"//char(48+is), 1, 0, output2D, mytype)
+          write(scname,"('phi',I2.2)") is
+          !call decomp_2d_register_variable(io_name, "phi"//char(48+is), 1, 0, output2D, mytype)
```

Comment:
The commented code should be removed

Reply:
I'll keep this comment for the moment just to have a reference to the olf way of creating the variable name. This is until we'll fix the issue with ADIOS2.